### PR TITLE
Fix tests to reflect new workqueue and user configs

### PR DESCRIPTION
**Pull request by cibelius**
## Description

Countryconfig PR: https://github.com/opencrvs/opencrvs-countryconfig/pull/1219

Fix tests to reflect new workqueues and user roles as per config sheets:
* [Workqueues](https://docs.google.com/spreadsheets/d/15y4B4s7byv1tIsXkxvcSwRK4YrqgBh_bfW1hjLAxh1I/edit?gid=0#gid=0)
* [Users](https://docs.google.com/spreadsheets/d/1hUsVXX8uAwEzsLzNW6PIltq0gFccURtbGRegVSxSmVM/edit?gid=1114347419#gid=1114347419)

## Checklist

- [x] I have linked the correct Github issue under "Development"
- [x] I have tested the changes locally, and written appropriate tests
- [x] I have tested beyond the happy path (e.g. edge cases, failure paths)
- [ ] I have updated the changelog with this change (if applicable)
- [x] I have updated the GitHub issue status accordingly

### CODE REVIEW

File: e2e/testcases/application/3-validate-sent-for-review.spec.ts
Comment (cibelius) [inline e2e/testcases/application/3-validate-sent-for-review.spec.ts:1]: This test file was removed as it's not longer relevant (Sent for review workqueue is removed)

File: src/api/workqueue/workqueueConfig.ts [hunk 1]
```diff
@@ -17,6 +17,20 @@ const DATE_OF_EVENT_COLUMN = {
...
+    location: user('primaryOfficeId')
+  }
+} as const
+
+const registeredInMyAdminArea = {
+  ['legalStatuses.REGISTERED.createdAtLocation']: {
+    type: 'within',
+    location: user('primaryOfficeId')
+  }
+} as const
```
Comment (naftis) [inline src/api/workqueue/workqueueConfig.ts:39]: Should we abstract these into toolkit for easy usage? Can be done later tho

Comment (cibelius) [reply src/api/workqueue/workqueueConfig.ts:39]: I believe there is also value in showing example usage of these kinds of filters which use `legalStatuses.<status>.createdAtLocation` and `location: user('primaryOfficeId')` here directly :)

**Review by naftis — APPROVED** [review]
